src/screens/WalletConnect/CallRequest/TransactionRequestContent.js

```diff
@@ -91,37 +105,77 @@ function TransactionRequestContent({ request, onConfirm, onReject }: Props) {
   const chainConfig = chainConfigs[chain];
   const value = wrapBigNumberOrNil(transactionPayload?.amount);
   const symbol = transactionPayload?.symbol;
-  const confirmTitle = !hasNotEnoughGas ? t('button.confirm') : t('label.notEnoughGas');
+  const confirmTitle = !hasNotEnoughGas ? t('button.swipeConfirm') : t('label.notEnoughGas');
   const isConfirmDisabled = isEstimating || hasNotEnoughGas || !!errorMessage;
 
   return (
     <>
-      <Text color={chainConfig.color}>
-        {title} {t('label.dotSeparator')} {chainConfig.titleShort}
+      <IconView>
+        <Icon name={chainConfig.iconName} width={64} height={64} style={styles.chainIcon} />
+        <ServiceIcon source={{ uri: iconUrl }} />
+      </IconView>
+
+      <Text color={colors.secondaryText} style={styles.body}>
+        {t('walletConnect.requests.transactionRequest')}
       </Text>
 
-      <Image source={{ uri: iconUrl }} style={styles.icon} />
+      <LargeFiatTokenValueView
+        value={value}
+        assetAddress={gasAddress}
+        chain={chain}
+        symbol={symbol}
+        style={styles.tokenValue}
+      />
 
-      <LargeTokenValueView value={value} symbol={symbol} style={styles.tokenValue} />
+      {!isConfirmDisabled && <TransactionDeploymentWarning chain={chain} style={styles.transactionDeploymentWarning} />}
 
       {!estimationErrorMessage && (
-        <FeeLabel
-          value={fee}
-          assetSymbol={gasSymbol}
-          assetAddress={gasAddress}
-          isLoading={isEstimating}
-          isNotEnough={hasNotEnoughGas}
-          style={styles.fee}
-          chain={chain}
-        />
+        <FeeView>
+          <FeeLabel
+            value={fee}
+            assetSymbol={gasSymbol}
+            assetAddress={gasAddress}
+            isLoading={isEstimating}
+            isNotEnough={hasNotEnoughGas}
+            chain={chain}
+            mode="actual"
+          />
+          <Icon name="info" width={16} height={16} color={colors.buttonTextTitle} style={styles.infoIcon} />
```

Comment:
We have this info icon which looks like a button, but it does not do anything. @paragramm what should happen when user presses it?
![image](https://user-images.githubusercontent.com/6368606/129885359-86063d73-79fc-47f9-b4ad-22b5fea3289a.png)